tests/unit/storage/models/storageModel.spec.ts

```diff
@@ -0,0 +1,45 @@
+import jsLogger from '@map-colonies/js-logger';
+import { StorageManager } from '../../../../src/storage/models/storageManager';
+import {
+  IStorageStatusResponse
+} from '../../../../src/common/interfaces';
+import { registerDefaultConfig } from '../../../mocks/config';
+import * as utils from '../../../../src/common/utils';
+
+let storageManager: StorageManager;
+
+describe('Storage', () => {
+  beforeEach(() => {
+    const logger = jsLogger({ enabled: false });
+    registerDefaultConfig();
+    storageManager = new StorageManager(logger);
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+    jest.restoreAllMocks();
+  });
+
+  describe('GetStorage', () => {
+    /**
+     * @deprecated GetMap API - will be deprecated on future
```

Comment:
why deprecated notation? if it was accidently copies from reference tests, remove the notation

Reply:
Removed.